<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test3Dialog</class>
 <widget class="QDialog" name="test3Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分子泵</string>
  </property>
  <property name="windowIcon">
   <iconset resource="errordetection.qrc">
    <normaloff>:/images/images/timg.jpeg</normaloff>:/images/images/timg.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分子泵电流</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>分子泵频率</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>54</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>分子泵状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="widget_4" native="true">
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>分子泵温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_test34">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>实时数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_test34">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="errordetection.qrc"/>
 </resources>
 <connections/>
</ui>
